setup.py

```diff
@@ -3,7 +3,7 @@
 # pylint: disable=consider-using-with
 setup(
     name="shell_gpt",
-    version="0.7.3",
+    version="0.7.4",
```

Comment:
Just a thought. If you wanted to avoid typing `r<Enter>` each time, you could instead go into revision mode immediately, but look out for special commands:

```
:e --- execute
:q --- quit
:* --- error message (: is always considered a command)
```

Disadvantage: You would not be able to send a reply to GPT starting with `:`